lib/agenda/index.ts

```diff
@@ -99,21 +116,21 @@ class Agenda extends EventEmitter {
   start!: typeof start;
   stop!: typeof stop;
 
-  constructor(config: any = {}, cb?: (error: Error, collection: Collection<any> | null) => void) {
+  constructor(config: AgendaConfig = {}, cb?: (error: Error, collection: Collection<any> | null) => void) {
```

Comment:
this was also something I was experimenting on.. as right now "from outside" it looks liek there ar eno default values, and you have to check the documetation of agenda again to find out.
another way of approach this , is to set default values in the constrcutor, check out https://github.com/hokify/agenda/blob/c5e21e98b92ef1ba77dab36f68a331c4b493b382/src/index.ts#L75 to see how I did it